tests/data/test_config.nconf

```diff
@@ -0,0 +1,16 @@
+NAILGUN:
```

Comment:
Break the nailgun and rizza configs into their own nconf files. They should lose their top-level self-naned entry since NanoConf will automatically add them as an attribute by their file name.